server/root/root.controller.ts

```diff
@@ -1,6 +1,6 @@
 import { Controller, Get, Req, Res } from "@nestjs/common";
 
-@Controller("api")
+@Controller()
```

Comment:
The health endpoint should still have the `api` prefix, otherwise it will break how `kubernetes` check for the service health in production.

Reply:
but with api prefix we run into SEO problem

![image](https://user-images.githubusercontent.com/73478823/163496594-7f9c056b-77f5-40bb-ab0b-7029b51a1e33.png)

Reply:
Let's put the prefix only for the health endpoint